osu.Game/Graphics/Backgrounds/Triangles.cs

```diff
@@ -177,7 +199,17 @@ protected virtual TriangleParticle CreateTriangle()
         /// Creates a shade of colour for the triangles.
         /// </summary>
         /// <returns>The colour.</returns>
-        protected virtual Color4 CreateTriangleShade() => Interpolation.ValueAt(RNG.NextSingle(), ColourDark, ColourLight, 0, 1);
+        protected virtual Color4 CreateTriangleShade(float shade) => Interpolation.ValueAt(shade, colourDark, colourLight, 0, 1);
```

Comment:
this should be moved inside the triangle itself, along with the random shade implementation.

Reply:
So remove that function completely?

Reply:
depends how it's being used in the `virtual` context, if at all.